### ISSUE TITLE
Add support for opening search result in new background tab

### ISSUE THREAD

**Issue by sgtpep** [bbeck/keyboard-shortcuts-for-google-search#1]
Very handy for mass-opening links. I hope I'll be able to provide pull request soon.

**Comment by bbeck** [comment]
I agree this would be a great feature to have.  I'd welcome any pull requests implementing this.

I've looked into this in the past and it turns out that content scripts aren't allowed to call most of the built-in chrome APIs (chrome.tabs.create would be the one we'd want to call).

There is a workaround though.  The extension would have to provide a background script that would listen to requests from the content script and then process them.  I've prototyped this and it looks like it works great.  Feel free to send over a pull request if you end up getting to it before me.

**Comment by sgtpep** [comment]
Yes, chrome.tabs.create doesn't work. But simple pop-under trick does. We should call `window.focus();` right after `window.open(...);` (tested on recent Chromium).

I'm not sure if it worth doing the whole pull request. Also I'd like to remap Shift+Enter to "open in background tab", and add Ctrl+Shift+Enter to "open in foreground tab".
